sporks.json

```
@@ -1309,6 +1309,45 @@
       }
     },
     "testnet": {
+      "testnet48": {
+        "id": 48,
+        "live": true,
+        "name": "Testnet48",
+        "sporkTime": "2023-10-25T15:21:04Z",
+        "rootHeight": "127720466",
+        "rootParentId": "2b30e75bd857f898456dcb296bea4b8bc8001cab7062eeee9e47876411b36d76",
+        "rootStateCommitment": "f2e5ebfca2fd519e49f7bd85bea81e92eeaa85a705b3376d8534e9c9649da710",
+        "gitCommitHash": "692969b1718b3d21f95ee7f66e5061623d99e599",
+        "stateArtefacts": {
+          "gcp": {
+            "rootCheckpointFile": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-48-execution/public-root-information/root.checkpoint",
+            "rootProtocolStateSnapshot": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-48/public-root-information/root-protocol-state-snapshot.json",
+            "rootProtocolStateSnapshotSignature": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-48/public-root-information/root-protocol-state-snapshot.json.asc",
+            "nodeInfo": "https://storage.googleapis.com/flow-genesis-bootstrap/testnet-48/public-root-information/node-infos.pub.json"
+          },
+          "s3": {
+            "rootCheckpointFile": "",
+            "rootProtocolStateSnapshot": "",
+            "nodeInfo": "",
+            "executionStateBucket": ""
+          }
+        },
+        "tags": {
+          "flow-go-tag": "v0.32.3",
+          "flow-dps-tag": "v0.32.3",
+          "cadence-tag": "v0.42.1",
+          "docker-tag": "v0.32.3"
+        },
+        "accessNodes": [
+          "access.devnet.nodes.onflow.org:9000"
+        ],
+        "seedNodes": [
+          {
+            "address": "access-003.devnet48.nodes.onflow.org:3570",
+            "key": "ba69f7d2e82b9edf25b103c195cd371cf0cc047ef8884a9bbe331e62982d46daeebf836f7445a2ac16741013b192959d8ad26998aff12f2adc67a99e1eb2988d"
```

this should be the original access-003 key.
```suggestion
            "key": "b662102f4184fc1caeb2933cf87bba75cdd37758926584c0ce8a90549bb12ee0f9115111bbbb6acc2b889461208533369a91e8321eaf6bcb871a788ddd6bfbf7"
```

---

I see. Changed, please check again